modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -9,1064 +9,400 @@ _Enterprise Edition_
-
-To obtain the registration token, you need to use the *Add Server* option in the Runtime Manager. This presents a complete command to register the Mule runtime engine in the format `./amc_setup -H <token> <server-name>`. After you have the command with the registration token, copy-paste it into the `$MULE_HOME/bin` folder for each Mule runtime engine you wish to register. Make sure to change the instance name `server-name` to the unique instance name you wish to use to label this Mule runtime engine in the Runtime Manager console.
-
-[NOTE]
-====
-You can use the same copied registration command for multiple Mule runtimes, but make sure to change the default instance name `server-name` to a different and unique instance name for each Mule runtime engine.
-====
-
-Here is an example `mule-agent.yml` file generated by the `-H` option:
-
-[source,yaml,linenums]
-----
-transports:
-  rest.agent.transport:
-    enabled: false
-  websocket.transport:
-    consoleUri: wss://mule-manager.anypoint.mulesoft.com:443/mule
-    handshake:
-      enabled: true
-      body:
-globalConfiguration:
-  security:
-    keyStorePassword: 42d9515f-3ca9-4ef4-87c0-586bd786b08b
-    keyStoreAlias: agent
-    keyStoreAliasPassword: 42d9515f-3ca9-4ef4-87c0-586bd786b08b
-  authenticationProxy:
-    endpoint: https://arm-auth-proxy.prod.cloudhub.io
-----
-
-[WARNING]
-====
-Do not register a single Mule runtime engine with multiple Runtime Manager business groups or environments.
-
-You can register each individual Mule with Runtime Manager only once.
-
-Do not register a Mule runtime engine with both an older xref:mule-management-console::index.adoc[Mule Management Console (MMC)] and Runtime Manager. If the Mule runtime engine is currently managed in MMC, first unregister the Mule runtime engine with MMC before running the `amc_setup -H` script.
-====
-
-[TIP]
-====
-MuleSoft support can provide you with some migration scripts to help you migrate from MMC to Runtime Manager.
-
-For details, see xref:managing-servers.adoc#add-a-server[Managing Servers].
-====
-
-=== Installation Via Proxy
+|
+|Encrypts a passphrase to use in the `mule-agent.yml` file.
 
-If your Mule runtime engine will run inside a firewall what restricts external communication through a proxy server, you can configure the Runtime Manager agent to route REST traffic through the proxy server to Runtime Manager.
+Use this option alone to generate an encrypted passphrase.
 
-To configure the proxy server, pass in the '-P' argument along with the -H argument.
+For information, see xref:rtm-agent-proxy-config.adoc#encrypt_password.
```

Comment:
I don't know if this is an issue from my Intellij interpreting this, but i see it as a plain text and not as a link
<img width="978" alt="Screen Shot 2019-10-07 at 10 44 49" src="https://user-images.githubusercontent.com/2117502/66317250-83593480-e8ef-11e9-84b7-11304e78321a.png">

Reply:
That's been fixed.

Reply:
I still see this one without the link, i checked in github and i also see it like that